workloads/router-perf-v2/common.sh

```diff
@@ -149,8 +178,12 @@ run_mb(){
 
 enable_ingress_operator(){
   log "Enabling cluster version and ingress operators"
-  oc scale --replicas=1 -n openshift-cluster-version deploy/cluster-version-operator
-  oc scale --replicas=1 -n openshift-ingress-operator deploy/ingress-operator
+  if "${HYPERSHIFT}"; then 
+    export KUBECONFIG="${HYPERSHIFT_MANAGEMENT_KUBECONFIG}"
+    oc scale --replicas=1 -n "${NAMESPACE}" deploy/cluster-version-operator
+    oc scale --replicas=1 -n "${NAMESPACE}" deploy/ingress-operator
+    export KUBECONFIG="${HYPERSHIFT_HOSTED_KUBECONFIG}"
+  fi
```

Comment:
I think if we leave it like this it will only run this for `hypershift`. it is also needed to run for a normal cluster, in my tests I used this code to achieve it:

```suggestion
  if [[ ${HYPERSHIFT} ]]; then
    export KUBECONFIG=${HYPERSHIFT_MANAGEMENT_KUBECONFIG}
    NAMESPACE="clusters-${CLUSTER_NAME}"
  else
    NAMESPACE="openshift-cluster-version"
  fi
  log "Enabling cluster version and ingress operators"
  oc scale --replicas=${NUMBER_OF_ROUTERS} -n ${NAMESPACE} deploy/cluster-version-operator
  oc scale --replicas=${NUMBER_OF_ROUTERS} -n ${NAMESPACE} deploy/ingress-operator
  if "${HYPERSHIFT}"; then KUBECONFIG="${HYPERSHIFT_HOSTED_KUBECONFIG}"; fi
```

It is basically using your logic from the previous steps to run those two `scale` commands